### ISSUE TITLE
Creating Custom Theme Question - Not Issue

### ISSUE THREAD

**Issue by kupoback** [atom/atom#7184]
There is a theme I use in Brackets that I love, however I prefer to use Atom over Brackets.

What is the easiest way to convert that over? I don't seem to 100% the mark-up of both. Obviously they will have different syntax, so I'm not sure what's the best way to try and convert it.

Any suggestions, please?

**Comment by mnquintana** [comment]
Is it a syntax theme or a UI theme?

**Comment by kupoback** [comment]
UI Theme

**Comment by kupoback** [comment]
![Brackets Theme](http://i.imgur.com/YrNXDYO.png)

Like this.

**Comment by simurai** [comment]
I would say best is to try to find a theme that comes close to the one you wanna port, and then fork that one and adjust it to match yours. Won't be 100% the same, but at least the colors could be similar. Maybe https://atom.io/themes/flatland-dark-ui

**Comment by kupoback** [comment]
I may have to do that. I am trying to figure out Atom's syntax and Bracket's syntax and compare them, but it's really difficult to break them both down. It's a Birds of Paradise from Coda 2, that is the only Theme that I really really enjoy. I can't find anything else one's created that comes close to pleasing to my eye. 

I'll have to spend a few hours figuring this out. I tried using Atom's default Theme package generator, but even those aren't clear, for me.

**Comment by simurai** [comment]
There is the https://atom.io/themes/birds-of-paradise-theme syntax theme. Not sure if that's exactly the same as Coda's version.

**Comment by kupoback** [comment]
Yeah, I've looked at that, and pulled it down, duplicated it and messed with it a bit, but I can't get the colours to match how it is in the image I showed above. It's quite a bit off of the Coda 2 version.

*Le sigh It'd be easier if I wasn't so picky, haha

I'm just trying to find something easy, dark, and classes, id's on CSS are different but not too vast, selectors are noticeable, values are noticeable. PHP the same, functions one, function names, selectors, markup, tags, etc instead of half being just white.
